sdk/Lusid.Sdk.Tests/tutorials/Instruments/InterestRateSwapExamples.cs

```diff
@@ -50,15 +85,33 @@ public void InterestRateSwapCreationAndUpsertionExample()
             Assert.That(roundTripSwap.StartDate, Is.EqualTo(swap.StartDate));
             Assert.That(roundTripSwap.Legs.Count, Is.EqualTo(swap.Legs.Count));            
             
-            // Delete Instrument
-            _instrumentsApi.DeleteInstrument("ClientInternal", uniqueId); 
+            // DELETE instrument
+            _instrumentsApi.DeleteInstrument("ClientInternal", uniqueId);
         }
         
         [TestCase(ModelSelection.ModelEnum.SimpleStatic)]
+        [TestCase(ModelSelection.ModelEnum.ConstantTimeValueOfMoney)]
+        [TestCase(ModelSelection.ModelEnum.Discounting)]
         public void InterestRateSwapValuationExample(ModelSelection.ModelEnum model)
         {
             var irs = InstrumentExamples.CreateExampleInterestRateSwap();
             CallLusidGetValuationEndpoint(irs, model);
         }
+        
+        [TestCase(ModelSelection.ModelEnum.ConstantTimeValueOfMoney)]
+        [TestCase(ModelSelection.ModelEnum.Discounting)]
+        public void InterestRateSwapInlineValuationExample(ModelSelection.ModelEnum model)
+        {
+            var irs = InstrumentExamples.CreateExampleInterestRateSwap();
+            CallLusidGetValuationEndpoint(irs, model);
+        }
+        
+        [TestCase(ModelSelection.ModelEnum.ConstantTimeValueOfMoney)]
+        [TestCase(ModelSelection.ModelEnum.Discounting)]
+        public void EquityOptionPortfolioCashFlowsExample(ModelSelection.ModelEnum model)
```

Comment:
Is this in the right file? Why is equity option in here?

Reply:
Another bad copy and paste ><"